modules/collection/programs/hypridle.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  config,
+  pkgs,
+  rumLib,
+  ...
+}:
+let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkEnableOption mkPackageOption mkOption;
+  inherit (rumLib.generators.hypr) toHyprconf;
+  inherit (rumLib.types) hyprType;
+
+  cfg = config.rum.programs.hypridle;
+in
+{
+  options.rum.programs.hypridle = {
+    enable = mkEnableOption "hypridle";
+
+    package = mkPackageOption pkgs "hypridle" { };
+
+    settings = mkOption {
+      type = hyprType;
+      default = { };
+      example = {
+        general = {
+          lock_cmd = "notify-send \"lock!\"";
+          unlock_cmd = "notify-send \"unlock!\"";
+          before_sleep_cmd = "notify-send \"Zzz\"";
+          after_sleep_cmd = "notify-send \"Awake!\"";
+          ignore_dbus_inhibit = false;
+          ignore_systemd_inhibit = false;
+        };
+
+        listener = [
+          {
+            timeout = 500;
+            on-timeout = "notify-send \"You are idle!\"";
+            on-resume = "notify-send \"Welcome back!\"";
+          }
+        ];
+      };
+      description = ''
+        Is written to `''${config.directory}/hypr/hypridle.conf`.
```

```suggestion
        Is written to {file}`$HOME/hypr/hypridle.conf`.
```

This is for future consistency with the documentation PR, but we do use [NixOS flavored CommonMark](https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#roles).

---

Oh yeah, can you check their other PRs with this?

---

Done!